src/unabto/unabto_common_main.h

```diff
@@ -91,6 +91,21 @@ void unabto_close(void);
 void unabto_tick(void);
 
 /**
+ * Initialize the nabto main setup to use AES cyptography.
+ * @param nms          The nabto main setup.
+ * @param preSharedKey Array of exactly 16Bytes containing the pre-shared key.
+ * @param pskLenght    The length of the pre-shared key.
+ * @return             Return true if the cryptography was successfully initialized.
+ */
+bool unabto_init_aes_crypto(nabto_main_setup* nms, uint8_t* preSharedKey, size_t pskLength);
```

Comment:
maybe the unabto_init name is a bit confusing now we have unabto_init, unabto_init_default_values and these. The crypto is actually default initialized so maybe it should be called set or setup